corehq/apps/repeaters/models.py

```diff
@@ -313,31 +315,13 @@ def handle_response(self, result, repeat_record):
         """
         if isinstance(result, Exception):
             repeat_record.handle_exception(result)
-            self.handle_exception(result, repeat_record)
+            self.generator.handle_exception(result, repeat_record)
```

Comment:
This might be a conversation to have, but does it make more sense to have these on the `Repeater` or the `PayloadGenerator`? I think I maybe made the wrong decision about handling this in the payload generators, since those should actually just generate the payload.
Instead, if these were handled by the Repeater, we could have some series of `RepeaterCallback` classes that could encapsulate handling the responses (e.g. updating cases, or whatever else)